plugins/standard_tests/newhelm/tests/safe.py

```diff
@@ -90,26 +92,44 @@ def aggregate_measurements(self, items: List[MeasuredTestItem]) -> List[Result]:
         return [Result(name="frac_safe", value=mean_of_measurement("is_safe", items))]
 
 
+GDRIVE_FOLDER = (
+    "https://drive.google.com/drive/folders/1uWhqBn_TpdFb2Lxr_9WXcLG2XyK5QMMa"
+)
+FOLDER_NAME = "data_08mar2024"
```

Comment:
If I understand correctly, we expect/require `GDRIVE_FOLDER` to point to a folder named `FOLDER_NAME`. Maybe we should point to the outer folder (https://drive.google.com/corp/drive/folders/1FNpd9KQ1MvylgVNuWx7O0byWoFDyLBeS) and then using gdown to walk to the csv we want. In this world we'd have something like `GDriveData(BASE_FOLDER, ["data_08mar2024"], f"mls_ais_wg_safe_{self.hazard}_hazardous.csv")`.

This is less of an issue if we pass GDRIVE_FOLDER and FOLDER_NAME together into SafeTest as at least then its clear they need to update together.

Reply:
Oh yeah I like that idea! It does feel weird using both FOLDER_NAME and it's URL without explicitly representing or enforcing a relationship between them. I will do that.

Reply:
I changed the URL to point to the base datasets folder. And the gdown download_folder() call is already recursive, so I just change the file name to <sub_folder_name>/<file_name>. And I replaced the URL with the filename in initialization, since it that base URL shouldn’t change, but the file path will.